README.md

```diff
@@ -74,32 +74,28 @@ export THREDDS_URL_ROOT=https://docker-dev03.pcic.uvic.ca/twitcher/ows/proxy/thr
 
 ### Installation
 
-Setup using virtual environment.
-Use Python 3 module `venv`, not `virtualenv`, which installs Python 2.
-Unit tests use `datetime.timezone`, which is available only in Python 3.
+Setup using `pipenv` which sets up the virtual environment automatically.
 
 ```bash
-$ python3 -m venv venv
-$ source venv/bin/activate
-(venv)$ pip install -U pip
-(venv)$ pip install -i https://pypi.pacificclimate.org/simple/ -r requirements.txt
-(venv)$ pip install -e .
+$ pip install -U pipenv
+$ pipenv install # --dev for development packages
```

Comment:
What does "for development packages" mean? Should this be explained in the README?

Reply:
I added a note in the readme. Basically things like `pytest` and `black` can be installed separately since they aren't actually needed to run the app.